config/creatrs/imp.cfg

```diff
@@ -61,242 +59,308 @@ Defence = 7
 ; It is ranged 0..100, and it's constant - doesn't change with experience level.
 Luck = 0
 ; Amount of health points regenerated every RecoveryFrequency turns when sleeping in lair.
-; Note that the amount is scaled with creature level - so always the same part of creature
-; health is regenerated. Default value of RecoveryFrequency is 10 turns.
+; Note that the amount is scaled with creature level - so always the same part of creature health is regenerated.
+; Default value of RecoveryFrequency is 10 turns.
 Recovery = 0
-; Amount of health points regenerated every RecoveryFrequency turns when Toking. It is
-; what creatures do without HealRequirement or LairSize, so Imps.
+; Amount of health points regenerated every RecoveryFrequency turns when Toking.
+; It is what creatures do without HealRequirement or LairSize, so Imps.
 TokingRecovery = 10
+; Amount of game turns before the creature gets hungry and needs to eat at the Hatchery.
 HungerRate = 0
+; Amount of food the creature must eat to stop being hungry and leave the Hatchery.
 HungerFill = 0
+; Defines how much capacity the lair totem of the creature will take on the lair.
 LairSize = 0
-; Amount of damage points inflicted when a creature walks on lava slabs.
+; Amount of damage points inflicted (every game turns) when a creature walks on lava slabs.
 ; Setting this to 0 will make the creature immune to lava.
 HurtByLava = 2
+; Defines the movement speed of the creature, range is 0..255.
 BaseSpeed = 96
+; Defines the maximum gold the creature can hold at a time.
 GoldHold = 500
-; Creature image size - XY and Z
+; Creature image size - XY and Z - not implemented.
```

Comment:
Actually, this is implemented and it is 'clipbox size'

Reply:
> Actually, this is implemented and it is 'clipbox size'

Ok, I removed "not implemented", thanks!